src/cmd/services/m3coordinator/downsample/options.go

```diff
@@ -648,6 +681,9 @@ func (cfg Configuration) newAggregator(o DownsamplerOptions) (agg, error) {
 		rs := rules.NewEmptyRuleSet(defaultConfigInMemoryNamespace,
 			updateMetadata)
 		for _, mappingRule := range cfg.Rules.MappingRules {
+			if strings.Contains(mappingRule.Filter, m3MetricsPrefixString) {
+				m3PrefixFilter = true
+			}
```

Comment:
nit: Want to do this in the rollup rules too? (see if they use the filter at all?)?

Reply:
sure, we do not do any specific handling for rollup rules yet, but would be good to do for completeness.